frontend/lib/config.ts

```diff
@@ -0,0 +1,25 @@
+
+export const requireEnv = (value: string | undefined, identifier: string) => {
+  if (!value) {
+    throw new Error(`Required env var ${identifier} does not exist`);
+  }
+  return value;
+};
+
+export const DEFAULT_CHAIN_ID = requireEnv(
+  process.env.NEXT_PUBLIC_DEFAULT_CHAIN_ID,
+  "NEXT_PUBLIC_DEFAULT_CHAIN_ID"
+);
+
+export const SUPABASE_URL = requireEnv(
+  process.env.NEXT_PUBLIC_SUPABASE_URL,
+  "NEXT_PUBLIC_SUPABASE_URL"
+);
+
+export const SUPABASE_ANON_KEY = requireEnv(
+  process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY,
+  "NEXT_PUBLIC_SUPABASE_ANON_KEY"
+);
+
+export const PLASMIC_PROJECT_ID = process.env.PLASMIC_PROJECT_ID ?? "MISSING";
```

Comment:
maybe wrap these in requireEnv so it crashes at build time when they're missing?

Reply:
Thanks for the comment! I think the issue is that this gets loaded client-side, and then they're missing because we don't have the NEXT_ prefix. In reality we only need them server-side for the build. I could either
1. move them into plasmic-init.ts 
2. keep this here, which will cause builds to fail since we're missing keys.